haterBackEnd/requirements.txt

```diff
@@ -1,6 +1,5 @@
 asgiref==3.5.1
 Django==4.0.4
```

Comment:
if ur going to delete the environment package then u should delete the comment in the settings.py lines 14-18